grammars.yml

```diff
@@ -541,9 +543,6 @@ vendor/grammars/marko-tmbundle:
 - text.marko
 vendor/grammars/mathematica-tmbundle:
 - source.mathematica
-vendor/grammars/matlab.tmbundle:
-- source.matlab
-- source.octave
```

Comment:
Some grammars might be embedding the `source.octave` scope, so I recommend adding a copy of `source.matlab` which aliases this. Even something as simple as a wrapper which embeds `source.matlab` using the `source.octave` scope.

~~~xml
<dict>
		<key>include</key>
		<string>source.octave</string>
</dict>
~~~

Reply:
Having said that, I haven't checked to see if there *are* grammars which use this scope, or whether Octave and MATLAB have any noticeable differences (I actually know nothing about either, save that Octave is a FOSS alternative).

YMMV.